README.md

```diff
@@ -41,3 +41,145 @@ Package can then be placed directly under `./charts/<centos|redhat>/<chart-name>
 Once this repository is available and configured, Helm Charts will be available in the [OpenShift Developer Perspective](https://docs.openshift.com/container-platform/latest/applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.html)
 
 You can also use Helm CLI commands, please refere to [Using Helm Guide](https://helm.sh/docs/intro/using_helm/) for detailed instructions on how to use the Helm client.
+
+## How to contribute to this repository
+
+In case of Helm chart is related to CentOS, CentOS Stream 8, or CentOS Stream 9 then use [centos](charts/centos) directory.
+
+In case of Helm chart is related to Red Hat then use [redhat](charts/redhat) directory.
+
+As soon as your Helm chart follows directory structure as mentioned above, let's test it.
+
+Follow the same guidelines for creating like they are specified here [openshift-helm-charts](https://github.com/openshift-helm-charts/charts/tree/main/docs).
+
+### General notes for Helm charts
+Each helm chart name has to contain file OWNERS. E.g. [OWNERS](./charts/redhat/postgresql-imagestreams/OWNERS)
+
+Each helm chart change, in [templates](./charts/redhat/postgresql-persistent/0.0.1/templates) or [OWNERS](./charts/redhat/postgresql-persistent/OWNERS) file
+has to increase version
+
+### How to test Helm chart
+
+For Helm chart testing you should have a connection to OpenShift 4 cluster and download and install oc binary.
+For access to OpenShift 4 cluster please ask phracek@redhat.com.
+
+#### Create a Helm chart package
+
+Before any testing proposes and deploying Helm chart to OpenShift cluster, you have to create a package by command:
+
+```commandline
+$ helm package <path_to_helm_chart>
+```
+
+E.g. in case of postgresql-persistent Helm chart you see message, like:
```

Comment:
Example: in the case of postgresql-persistent Helm chart the output of the command looks like this: